ChangeLog.md

```diff
@@ -1,5 +1,171 @@
 # Changelog for haskell-language-server
 
+## 0.9.0
+
+This release includes lot of refactorings and bug fixes over existing features, hlint and eval plugins among others.
```

Comment:
An important bug with diagnostics has also been fixed, I think it could be mentioned too.

Reply:
Uh, sorry, i am not finding it right now, what would be the issue/pr?

Reply:
https://github.com/haskell/haskell-language-server/pull/1204